### PR TITLE
Resize Frame filter

### PR DESCRIPTION
**Pull request by sujayjaju**
Ability to resize frame - to create custom sized thumbnails / image outputs

### CODE REVIEW

**Comment by beesofts** [comment]
need this feature too

**Comment by karpy47** [comment]
Please merge this commit! I need it also.

File: src/FFMpeg/Filters/Frame/ResizeFilter.php [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        $dimensions = null;
+        $commands = array();
+
+        foreach ($frame->getVideo()->getStreams() as $stream) {
+            if ($stream->isVideo()) {
+                try {
+                    $dimensions = $stream->getDimensions();
+                    break;
+                } catch (RuntimeException $e) {
+
```
Comment (patkar) [inline src/FFMpeg/Filters/Frame/ResizeFilter.php]: A silent catch is never good. Would do you think you want to catch?

**Comment by patkar** [comment]
@sujayjaju May you add unit tests please?

**Comment by Romain** [comment]
@romainneutron Shouldn't we close this PR?
I think we already have a PR filter...

**Comment by aliabbas-2012** [comment]
You haven't done the Resize frame filter

**Pull request closed without merging**